account_bank_statement_import_ofx/wizard/account_bank_statement_import.py

```diff
@@ -65,30 +65,23 @@ def _parse_file(self, data_file):
 
         transactions = []
         total_amt = 0.00
-        start_date = end_date = False
         try:
             for transaction in ofx.account.statement.transactions:
                 vals = self._prepare_ofx_transaction_line(transaction)
                 if vals:
                     transactions.append(vals)
                     total_amt += vals['amount']
-                    tdate = fields.Date.to_string(vals['date'])
-                    if not start_date or tdate < start_date:
-                        start_date = tdate
-                    if not end_date or tdate > end_date:
-                        end_date = tdate
         except Exception, e:
             raise UserError(_(
                 "The following problem occurred during import. "
                 "The file might not be valid.\n\n %s") % e.message)
 
+        balance = float(ofx.account.statement.balance)
         vals_bank_statement = {
-            'name': _('Account %s %s > %s') % (
-                ofx.account.number, start_date, end_date),
+            'name': ofx.account.number,
```

Comment:
Over at the CAMT corner, we are really struggling with statement identifiers that are recycled every year. Are you sure that is not the case for OFX?